examples/mlmirror.py

```diff
@@ -20,11 +27,12 @@
 					  [0, 1, 0],
 					  [-1, 0, 0]])
 
-source = LabConstantPointSource([100., 0., 0.], -1., 1., 0.35, direction='-x')
+#source = LabConstantPointSource([100., 0., 0.], -1., 1., 0.35, direction='-x')
```

Comment:
This comment lines can be removed - even if you want to go back to a constant energy, it would be better to wrote `energy=1.` (I may have even made that a requirement when I changed the code for sources.)